src/colvar_rotation_derivative.h

```diff
@@ -3,6 +3,7 @@
 
 #include "colvartypes.h"
 #include <type_traits>
+#include <cstring>
```

Comment:
Not sure why this is included.

Reply:
I use `std::memset` and `std::memcpy` below:
https://github.com/Colvars/colvars/pull/585/files/7d4ac3229dc213cce5ccce8497d5856301135466#diff-2bb16194d8ff7f05ebd64436eadfccfcd11fc1195dc54b56367154554b198ab1R595-R597

Reply:
Thanks! It's very un-intuitive to me that those should be declared in a header named cstring.h.

Reply:
@jhenin It wouldn't C++ if it made sense all the time, would it? :-)